src/stratis_cli/_parser/_parser.py

```diff
@@ -142,7 +142,10 @@ def wrapped_func(*args):
                 (
                     "report_name",
                     dict(
-                        default=None, type=str, help=("Name of the report to display")
+                        default="engine_state_report",
+                        type=str,
+                        help=("Name of the report to display"),
+                        nargs="?",
```

Comment:
Please add a `choices` parameter here as well with the two available names, ```engine_state_report``` and ```errored_pool_report```.